### PR TITLE
cryptsetup: drop redundant parens

### PR DESCRIPTION
**Pull request by poettering**

### CODE REVIEW

File: src/cryptsetup/cryptsetup.c [hunk 1]
```diff
@@ -1988,7 +1988,7 @@ static int run(int argc, char *argv[]) {
                 if (until == USEC_INFINITY)
                         until = 0;
 
-                arg_key_size = (arg_key_size > 0 ? arg_key_size : (256 / 8));
+                arg_key_size = arg_key_size > 0 ? arg_key_size : 256U / 8U;
```
Comment (keszybz) [inline src/cryptsetup/cryptsetup.c]: I think this should just use a normal `if`.

**Comment by poettering** [comment]
updated as suggested

**Comment by keszybz** [comment]
LGTM. (I think the `U`s a re a bit overkill, but meh.)

File: src/cryptsetup/cryptsetup.c [hunk 2]
```diff
@@ -1988,7 +1988,8 @@ static int run(int argc, char *argv[]) {
                 if (until == USEC_INFINITY)
                         until = 0;
 
-                arg_key_size = (arg_key_size > 0 ? arg_key_size : (256 / 8));
+                if (arg_key_size == 0)
+                        arg_key_size = 256U / 8U;
```
Comment (daandemeyer) [inline src/cryptsetup/cryptsetup.c:1992]: Bit off topic but we should be using larger keys these days supposedly, see https://github.com/systemd/systemd/pull/24944#discussion_r994390109

Comment (poettering) [reply src/cryptsetup/cryptsetup.c:1992]: 256bit should be OK for now. Note that this is only used for superblock-less crypto volumes. If you do LUKS you pick the defaults in the cryptsetup luksFormat command line.

**Review by bluca — APPROVED** [review]

**Pull request merged**